interface/patient_file/summary/demographics.php

```diff
@@ -29,7 +29,7 @@
  * @package LibreEHR
  * @author Brady Miller <brady@sparmy.com>
  * @author Terry Hill <teryhill@librehealth.io>
- * @link http://www.libreehr.org
```

Comment:
Changed that commit on the way. I have this code running on a customers site . That is how I found the problem originally.